src/main/java/com/onlinebookstore/model/Book.java

```diff
@@ -27,5 +31,8 @@ public class Book {
     @Column(nullable = false)
     private BigDecimal price;
     private String description;
+    @Column(name = "cover_image")
```

Comment:
Remove all redundant @Column(name = "....) 
Hibernate create column name in snake case automatically